lib/charms/layer/jenkins/api.py

```diff
@@ -175,7 +194,10 @@ def _execute_action(self, action, fail_message):
         client = self._make_client()
         request = requests.Request("POST", urljoin(self.url, action))
         try:
-            client.jenkins_open(request)
+            # Jenkins doesn't return an error for some actions
+            if (client.jenkins_open(request) and
+                (action == "quietDown" or action == "cancelQuietDown")):
```

Comment:
Would this work the same if reversed? If so it sounds like that might be preferable as it would avoid running client.jenkins_open(request) if action isn't "quietDown" or "cancelQuietDown".

Reply:
Can you confirm if this wouldn't work if reversed? i.e.

`if ((action == "quietDown" or action == "cancelQuietDown") and client.jenkins_open(request)):`